core/src/main/scala/kafka/tools/StorageTool.scala

```diff
@@ -139,7 +139,7 @@ object StorageTool extends Logging {
       action(storeTrue())
     formatParser.addArgument("--release-version", "-r").
       action(store()).
-      help(s"A KRaft release version to use for the initial metadata version. The minimum is 3.0, the default is ${MetadataVersion.LATEST_PRODUCTION.version()}")
+      help(s"A KRaft release version to use for the initial metadata.version. The minimum is ${MetadataVersion.IBP_3_0_IV0}, the default is ${MetadataVersion.LATEST_PRODUCTION.version()}")
```

Comment:
I saw 2 formats: `MetadataVersion.IBP_3_0_IV0` and `MetadataVersion.IBP_3_0_IV0.version()` here. Could you help me understand what output it will be for each of them? And I think we should make them consistent. Same as other places.

Reply:
Hi @showuon, thanks for the review. Using `MetadataVersion.IBP_3_0_IV0` or `MetadataVersion.IBP_3_0_IV0.version()` has same result. Both `toString` and `version` functions return `ibpVersion`:

https://github.com/apache/kafka/blob/58ddd693e69599b177d09c2e384f31e7f5e11171/server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java#L523-L525
https://github.com/apache/kafka/blob/58ddd693e69599b177d09c2e384f31e7f5e11171/server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java#L632-L634

Reply:
OK, thanks for the explanation. Could we at least make them consistency (all with `.version()` or all without it)?

Reply:
Yes, it looks like we only use `.version()` in few messages, so I remove it. Thank you.